ios/PhyWeb/Backend/PWBeaconManager.m

```diff
@@ -113,19 +117,20 @@ - (void)unregisterConfigurationChangeBlock:(id)registeredBlock {
 }
 
 - (void)start {
+  _startTime = [NSDate timeIntervalSinceReferenceDate];
   _started = YES;
   PWBeaconManager* __weak weakSelf = self;
   _scanner = [[UBUriBeaconScanner alloc] init];
   [_scanner startScanningWithUpdateBlock:^{
-      PWBeaconManager* strongSelf = weakSelf;
-      [strongSelf _updateBeacons];
+    PWBeaconManager* strongSelf = weakSelf;
+    [strongSelf _updateBeacons];
   }];
   _httpServiceBrowser = [[NSNetServiceBrowser alloc] init];
   [_httpServiceBrowser setDelegate:self];
   _httpsServiceBrowser = [[NSNetServiceBrowser alloc] init];
   [_httpsServiceBrowser setDelegate:self];
-  [_httpServiceBrowser searchForServicesOfType:@"_http._tcp." inDomain:nil];
-  [_httpsServiceBrowser searchForServicesOfType:@"_https._tcp." inDomain:nil];
+  [_httpServiceBrowser searchForServicesOfType:@"_http._tcp." inDomain:@""];
```

Comment:
Yes. That was a warning fix. The second parameter of -searchForServicesOfType:inDomain: can't be nil. The correct value is an empty string.